docker/cami-medical-compliance/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM williamyeh/ansible:ubuntu16.04
```

Comment:
@iprunache I'm a bit surprised that we're not using the 'official' ubuntu images (https://hub.docker.com/_/ubuntu/), so I'm guessing this must be something related to Ansible support. If that's the case, a short comment would be great!